src/app/actions/rsvp-group.js

```diff
@@ -49,7 +68,37 @@ export const fetchRsvpGroup = (code, showModal = true) => {
   }
 
   return dispatch => {
-    dispatch(requestRsvpGroup());
-    return setTimeout(fetchData.bind(this, dispatch), 100);
+    dispatch(getRsvpGroup());
+    return setTimeout(getData.bind(this, dispatch), 100);
   };
 }
+
+export const submitPostRsvpGroup = (userGroup, users) => {
+  const { id } = userGroup;
+  const POST_URL = postRsvpGroupUrl(id);
+
+  function postData(dispatch) {
+    axios({
+      url: POST_URL,
+      method: 'put',
+      data: sanitizePostData({ userGroup, users }),
+      headers: {
```

Comment:
Should look into a middleware solution or configuration to handle these defaults.